### ISSUE TITLE
Restrict guests at Tournaments

### ISSUE THREAD

**Issue by Atyrat** [Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2#1217]
**Describe your suggestion in full detail below:**
TL;DR If a human kingdom holds a tournament, most of the attendees should be humans/friendly races and religions, not mostly races that humans probably don't even consider as people.

I was trying out the new update playing as Jaina, and decided to see how the tournament mechanics all fit in. When I started my tournament, I saw that my guests included murlocs, harpies, ogres, centaurs, even some satyr! I do think having a few weird attendees would be fun, and there are friendly npcs of those races, but obviously, a festival put on by a human kingdom would not be inviting many demons and harpy leaders.  I think some restriction needs to be put in place where for a character with a position at any other court to attend your events, they need to be either of a non-evil religion, or have their liege be in an alliance with you. 

For wandering courtiers and such, I think they should be able to visit events even if they have an evil religion, but that evil religion should be a big negative modifier, so it doesn't happen constantly.
